pkg/logs/input/kubernetes/launcher.go

```diff
@@ -218,6 +243,14 @@ func (l *Launcher) getAnnotation(pod *kubelet.Pod, container kubelet.ContainerSt
 	return ""
 }
 
+//getServiceLabel returns the standard service label for container if present
+func (l *Launcher) getServiceLabel(pod *kubelet.Pod, container kubelet.ContainerStatus) string {
+	if serviceLabel, exists := pod.Metadata.Labels[podStandardLabelService]; exists {
+		return serviceLabel
+	}
```

Comment:
you can have a second case here with the label:
`tags.datadoghq.com/<container-name>.service`
so I think you need to check both labels

Reply:
the label with the container name take precedence on the other label.

Reply:
done